spec/components/fetch_month_component_spec.rb

```diff
@@ -21,7 +21,7 @@
         end
 
         it {
-          is_expected.to eq 'success: Created <a target="_new" href="https://argo.stanford.edu/view/druid:1234">druid:1234</a>'
+          expect(subject).to eq 'success: Created <a target="_new" href="https://argo.stanford.edu/view/druid:1234">druid:1234</a>'
```

Comment:
this just traded `RSpec/ImplicitSubject` for `RSpec/NamedSubject`. Is that better? Maybe we should just name the subject and have zero violations?